package.json

```diff
@@ -36,6 +36,7 @@
 		"@typescript-eslint/eslint-plugin": "^4.28.2",
 		"@typescript-eslint/parser": "^4.28.0",
 		"cz-conventional-changelog": "^3.3.0",
+		"discord-api-types": "^0.18.1",
```

Comment:
Why is this at the global level?

Reply:
Because all dev deps are. It's a sneaky way to present a cleaner bundle on npm.